test/jdk/com/sun/crypto/provider/KeyAgreement/SupportedDHParamGensLongKey.java

```diff
@@ -24,6 +24,7 @@
 /**
  * @test
  * @bug 8072452 8163498
+ * @library /test/lib
```

Comment:
It looks like you can remove this change & file from the PR

Reply:
The reason for this, is that this test is calling another : SupportedDHParamGens which uses the SecurityUtils, it does need to build /test/lib to compile